articles/020_ros_with_dds.md

```diff
@@ -139,9 +147,17 @@ OpenSplice appears to be the number two DDS implementation in use, but that is h
 
 TwinOaks's CoreDX DDS implementation is proprietary only, but apparently they specialize in minimal implementations which are able to run on embedded devices and even bare metal.
 
+eProsima's FastRTPS implementation is available on GitHub and is LGPL licensed:
+
+[https://github.com/eProsima/Fast-RTPS](https://github.com/eProsima/Fast-RTPS)
+
+eProsima Fast RTPS is a relatively new, lightweight, and open source implementation of RTPS.
+It allows direct access to the RTPS protocol settings and features, which is not always possible with other DDS implementations.
+eProsima's implementation also includes a minimum DDS API, IDL support, and automatic code generation and they are open to working with the ROS community to meet their needs.
+
 Given the relatively strong LGPL option and the encouraging but custom license from RTI, it seems that depending on and even distributing DDS as a dependency should be straight forward.
 One of the goals of this proposal would be to make ROS 2.0 DDS vendor agnostic.
-So, just as an example, if the default implementation is RTI, but someone wants to use OpenSplice, they simply need to recompile the ROS source code with some options flipped and they could use the OpenSplice implementation.
+So, just as an example, if the default implementation is Connext, but someone wants to use one of the LGPL options like OpenSplice or eProsima Fast RTPS, they simply need to recompile the ROS source code with some options flipped and they can use the implementation of their choice.
```

Comment:
Since all three vendors and products have been mentioned above and Connext and OpenSplice are only mentioned as product names in this sentence I would also only say `FastRTPS` (without the vendor as well as without the space).